packages/tc-web/src/components/tc-grid-container/tc-grid-container.tsx

```diff
@@ -0,0 +1,100 @@
+import { Component, Prop, h } from "@stencil/core";
+
+@Component({
+  tag: "tc-grid-container",
+  shadow: true,
+  styleUrl: "tc-grid-container.css",
+})
+export class TcGridContainer {
+  @Prop() mdColumns: number = 2;
+  @Prop() lgColumns: number = 4;
+  @Prop() columns: number = 1;
+  @Prop() gap: string = "";
```

Comment:
maybe like

```suggestion
  @Prop() gap: 'sm' | 'md' | 'lg' | 'xl' = "md";
```

using a default of md to cover the "default case" in the switch statement (so that default case could be removed now)